data/autoyast_sle15/pc_tools.xml

```diff
@@ -1,146 +1,144 @@
-            </addon>
-        </addons>
-    </suse_register>
-    <bootloader>
-        <device_map config:type="list">
-            <device_map_entry>
-                <firmware>hd0</firmware>
-                <linux>/dev/vda</linux>
-            </device_map_entry>
-        </device_map>
-        <global>
-            <timeout config:type="integer">1</timeout>
-            <append>splash=verbose</append>
-        </global>
-    </bootloader>
-    <networking>
-        <keep_install_network config:type="boolean">true</keep_install_network>
-    </networking>
-    <software>
-        <products config:type="list">
-            <product>SLES</product>
-        </products>
-         <packages config:type="list">
-            <package>netcat-openbsd</package>
-            <package>git-core</package>
-            <package>ntp</package>
-            <package>gcc</package>
-            <package>sudo</package>
-            <package>wget</package>
-            <package>python</package>
-            <package>haveged</package>
-        </packages>
-    </software>
-    <scripts>
-        <chroot-scripts config:type="list">
-            <script>
-                <chrooted config:type="boolean">true</chrooted>
-                <interpreter>shell</interpreter>
-                <source>
-                    <![CDATA[
-                        #!/bin/sh
-                        update-alternatives --install /usr/bin/python python /usr/bin/python2 1
-                        update-alternatives --install /usr/bin/python python /usr/bin/python3 2
-                        ]]>
-                </source>
-            </script>
-        </chroot-scripts>
-    </scripts>
-    <partitioning config:type="list">
-        <drive>
-            <device>/dev/vda</device>
-            <disklabel>msdos</disklabel>
-            <initialize config:type="boolean">true</initialize>
-            <partitions config:type="list">
-                <partition>
-                    <mountby config:type="symbol">device</mountby>
-                    <filesystem config:type="symbol">swap</filesystem>
-                    <mount>swap</mount>
-                </partition>
-                <partition>
-                    <mountby config:type="symbol">device</mountby>
-                    <filesystem config:type="symbol">ext4</filesystem>
-                    <mount>/</mount>
-                </partition>
-            </partitions>
-            <use>all</use>
-        </drive>
-    </partitioning>
-    <users config:type="list">
-        <user>
-            <fullname>Bernhard M. Wiedemann</fullname>
-            <encrypted config:type="boolean">false</encrypted>
-            <user_password>nots3cr3t</user_password>
-            <username>bernhard</username>
-        </user>
-        <user>
-            <encrypted config:type="boolean">false</encrypted>
-            <user_password>nots3cr3t</user_password>
-            <username>root</username>
-        </user>
-    </users>
+  <general>
```

Comment:
not sure why similar lines are marked in the diff . it is hard to see what actually was changed ...

Reply:
Looks like the 4 spaces were converted into 2 spaces, same as all the other profiles we have.
Anyway, the main difference is these 2 lines:
```
<ask_user config:type="boolean">false</ask_user>
<selected config:type="boolean">true</selected>
```
which are not compatible with 15-SP3.

Reply:
As suggestion of @asmorodskyi I split the indentation cosmetic in an other commit to simplify the review